src/components/ModeMenu.js

```diff
@@ -0,0 +1,14 @@
+import ModeMenuButton from "./ModeMenuButton";
+
+const ModeMenu = (onButtonClick) => {
+    const section = document.createElement('div');
+    section.classList.add('modeMenu');
+
+    ['Characters', 'Quotes', 'Deaths'].map((buttonName)=>{
+      let newButton = ModeMenuButton(buttonName,onButtonClick);
+      section.appendChild(newButton)});
```

Comment:
Funkcja map, służy do tworzenia jednej tablicy na podstawie innej.
Ty tutaj wykorzystujesz tą funkcję ale jej do niczego nie przypisujesz, więc moim zdaniem użycie jej jest nietrafione. 
Jeżeli chcemy przelecieć po elementach tablicy i wykonać jakąś akcję dla każdego elementu, to powinniśmy użyć pętli forEach.
Dlatego proponuję coś takiego:
```suggestion
    ['Characters', 'Quotes', 'Deaths'].forEach(buttonName => {
      section.appendChild(ModeMenuButton(buttonName, onButtonClick));
    });
```

Reply:
@PSwiecicki 
Nie koniecznie. To wszystko zależy. W tym przypadku chyba też użylbym funkcji map, gdyż jest o wiele szybsza niż funkcja forEach.

Ani to ani to nie będzie to błędem.
Ja użylbym też map.

Ciekawy i krótki link

https://codeburst.io/javascript-map-vs-foreach-f38111822c0f

Reply:
W takim razie przerobił bym to bardziej na 

```suggestion
    const categoriesButtons = ['Characters', 'Quotes', 'Deaths'].map( buttonName => ModeMenuButton(buttonName, onButtonClick));
    section.append(categoriesButtons);
```

Reply:
Ej, nie mamy jeszcze tak zaawansowanego JS'a :D
To może jeszcze tak? :DD
```suggestion
    section.append(['Characters', 'Quotes', 'Deaths'].map( buttonName => ModeMenuButton(buttonName, onButtonClick)));
```

Reply:
Agnieszka, tutaj wydziwiamy z Patrykiem. 
Myślę, że tak jak zrobiłaś, jest całkiem fajnie i może zostać!

Reply:
Ta druga opcja Patryka mnie przekonała :)